src/HasBinaryUuid.php

```diff
@@ -6,6 +6,9 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Database\Eloquent\Builder;
 
+/**
+ * @property string|null $uuid_text
```

Comment:
This docblock should be removed as `$uuid_text` isn't always the name of the textual property.

Reply:
I see what you mean :+1: